docs/introduction.md

```diff
@@ -14,8 +14,9 @@ This guide is here to serve a community as helpful content on the Neo4j journey.
 In this guide you can find tips and ticks for following topics: 
```

Comment:
Line 9: Learning and exploring the database can be fun, but you can sometimes run into some **wired** performance or scale issues. 

weird